.gitignore

```diff
@@ -0,0 +1,3 @@
+bin
+obj
+**/.idea
```

Comment:
add in a `**/secrets.json` too
also i tend to use https://gitignore.io to generate one, the languages + editors (give me a shout if you need a hand)